python_src/src/lamp_py/migrations/versions/performance_manager_staging/005_96187da84955_remove_metadata.py

```diff
@@ -0,0 +1,67 @@
+"""remove_metadata
+
+Revision ID: 96187da84955
+Revises: 45dedc21086e
+Create Date: 2023-12-28 12:18:25.412282
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "96187da84955"
+down_revision = "45dedc21086e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Actually, we need to check if records exist in the new metadata log table before executing the drop table command, otherwise the table could be dropped before the records are transferred to the new metadata_log table.